scripts/src/public/services/users.ts

```diff
@@ -14,6 +14,7 @@ import { create as createUserRegistrationRequested } from "../../analytics/event
 import { create as createUserRegistrationSucceeded } from "../../analytics/events/user_registration_succeeded";
 import { create as createUserLogoutServerRequested } from "../../analytics/events/user_logout_server_requested";
 import { create as createWalletAddressUpdateSucceeded } from "../../analytics/events/wallet_address_update_succeeded";
+import { create as createRestoreRequestFailed } from "../../analytics/events/restore_request_failed";
```

Comment:
where's this file?
it's not part of the PR, and i don't see it in master

Reply:
Yeah, adding the file will benefit the PR immensely

Reply:
@nitzantomer thanks